src/pages/docs/cli/workers.mdx

```diff
@@ -269,3 +270,24 @@ For example to look for oplog entries that contain parameters or return values o
 ```bash
 golem-cli worker oplog --worker urn:worker:d8bc9194-a4a2-4a57-8545-43408fc38799/counter1 --query 'exported-function AND (product-id:123 OR product-id:456)'
 ```
+
+## Reverting a worker
```

Comment:
Does it make sense to explicitly point out that side effects performed as part of the invocations will not be reverted?